meta-mender-core/recipes-mender/mender-shell/mender-shell.bb

```diff
@@ -20,6 +20,7 @@ RDEPENDS_${PN} = "glib-2.0"
 MENDER_SERVER_URL ?= "https://docker.mender.io"
 MENDER_SHELL_USER ??= "nobody"
 SYSTEMD_AUTO_ENABLE ?= "enable"
+SYSTEMD_SERVICE_${PN} = "mender-shell.service"
```

Comment:
Got it, thanks.

We should then remove these lines from below, right?
```
FILES_${PN}_append += "\
    ${systemd_unitdir}/system/mender-shell.service \
"
```

Reply:
I think that also needs to be there. `FILES` is required to include the file in the package. `SYSTEMD_SERVICE` is for marking this particular file as a service to be enabled.

Reply:
You are right (as usual). I got confused as I couldn't grep for this in `mender-client` equivalent recipe.

Not important, but if you have to rework anyway,

I suggest then that we move the `FILES_${PN}_append` up here, next to the `SYSTEMD_SERVICE_${PN}`, and possibly add any other missing files (the conf file is added at `do_install` but not explicitly included in `FILES_${PN}`) in a way that this recipe is closer to the [mender-client one](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-core/recipes-mender/mender-client/mender-client.inc#L30).